tests/fixtures/vcr_cassettes/get-records-two-errors.yaml

```diff
@@ -83,54 +96,27 @@ interactions:
       Connection:
       - keep-alive
       User-Agent:
-      - python-requests/2.31.0
+      - python-requests/2.32.3
     method: GET
     uri: https://dspace.mit.edu/oai/request?identifier=oai%3Adspace.mit.edu%3A1721.1%2F152786&metadataPrefix=oai_dc&verb=GetRecord
   response:
     body:
-      string: !!binary |
-        H4sIAAAAAAAAA+1ZzW4bNxA+N0/BuocogE1pV5JlK4oANbZjH1wbloMECHKguJSWNZfckFxLrmGg
-        79A37JN0uNzVT22pcpocWuxBgnY4883HIXf4Qez9GClq71KGYpuIfs99I0Hk5M0OkzvwzEjU71lu
-        BeufXl9foqElNjOo3WigP3//A51Jy7QkAg2ZvmUaHWutdK/uA3rG3gmGHPybHctmtk6N2emPVHSH
-        7sdK2r0xSbi4616TWCVkd6A5EbuGSLNnmObj1w8oDnZRHMKnuYtG6J4qoXR3GnPLXo8IvZlolclo
-        z5t/aofto85+HlXgG/4b64ZhOnPGcNkY7Htjc8XYyo3pis1HkzL5SEBeMGDBJUP3MeOT2HYDcFpL
-        aKR0xHRXKsleP/TqeVX6vbqvrasG1Dl4TnnBuxdrRAUx5s2OI7KD6v1eCmj9a6h2rz7qo+MZZanl
-        SqIrlipte/W0cDlnxpCJ96IqweJOUhUxPFOE44hYkmp1y4EyZiWGwR80t1xOPiZiDtz1fJDVdzCC
-        rEIqs2lm0cuXi2RHzFDNc/884XXMkPGzYZA2czNkESISZZLNUkYtPFElI55ztzGxKNXslkk3wC0a
-        a5WgcSbGXIg8LQBq9iVjxuJF2jnJPGkPEPq/kluCucJnF0sz+E7zf/GD0hMcmZTQAtfNmVNmME9S
-        gSmhMcNHQzf+8WJw9tY9D70LNlZpVlsz6GbRDYLmq8cpoGpWKyGAuA+G2COYzmU5HecAr+GAOva1
-        p308fqMF+CaTWLOxgDXB75hkmsASnDMbq2hAYSpG6VYYYC5v1Q2rvZc3Uk0lGqpMU/a38CMm2IS4
-        Aq7Gn7liFAAbfXJaLTfrfBldi5qDe/cSpnjyAYcHRZ1MqgF4rEnCpkrf4Ckb4cR7mix17wc+g3hK
-        RoKdEhlBEVdh14zmacKg/Y3SnCh95ffyxoRBM9yQ0G01wSxObmmZnEipoLXAsuOhH91IYyAjb65t
-        9l7aLF9HppjsOUlTiCzABxFJoSsUXLzRJ6xt9M/ZdFrN78Mmzh/LjrwFk/3OMxdpMDJWE2pXarya
-        vrbRJ097sE3WIw59w0Jb0cUC40iVttrjwRz4sLlNZZ9CLnrXOuCD/c4WwCeltYyFLu0aRPm+PBr3
-        pNvB12BH6h1bB3nQCos2NJsDxNam+BS+Ss+i3deWbX5XhF9FqMRbQyns/AtKnbDskiTNzyWrEkqs
-        I2QUvSlCYX99MCdcuI0WKf+jNrfkQG0oDBzB2qmb8iC+UsqityQzbHESf7dDdzPwLBH4mMbqWLAE
-        5ASeAjqrLVv8LFw1/hkJjGkSu8OHuDGnuwrEFZOv8LMgrxgFxbgEuGQo2n/jOXiwlR9Brto8ySdB
-        M8sFhgU4u3gPv4wPL+pVW7L7g/BZvEB6XJ6f2gWpJYOHy3vZM/FWsAoZ0HpCLX1TQXbYqfRYpccq
-        PVbpsUqPVXrsv6jHqKV4auzMyS8AtrPji5Ml0fV+8ecMDCAuESy3UJMXiFj0SavpLlUC3Wf+0Nkz
-        +aHz8LmLPgW7jc/+EJ+nMBoPrWYkGVJ424GtjQFhkQIy1FYdihe48+ox0s/EcOq9rxiJ5q+mA3k8
-        VpwPwf5WSMDInmiVXOZzXQvXCMOt4daBBEFQtm2ndGJYIdAdZBZizaFa8OPY/QfmQ/Ij2KHlttqT
-        w55au72VgtpCFjeDShZXsriSxZUsrmRxJYsrWVzJ4v+TLP5FWTa/nxxnQiAQX/QGuV3MkBr7a0an
-        nanTzogbBFBcuGbj1LBdXGuCUjT+JvKJO9q42R/k7NF1zr7ewQ0cNA57dRgBAvltcD2/jP8Ly9YB
-        6pwfAAA=
+      string: ""
```

Comment:
I did manually set the `500` as well because that handle is now returning a `200` with a record. Agree with your point about creating a fake response string being confusing and inaccurate. It did ultimately feel like too much overhead to support both `vcrpy` and `requests-mock` in the test suite but as you say, we can evaluate this more comprehensively later